### PR TITLE
springer-webfonts: first major release

### PR DESCRIPTION
**Pull request by davidpauljunior**

### CODE REVIEW

File: toolkits/springer/packages/springer-webfonts/HISTORY.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 1.0.0 (2019-02-10)
```
Comment (nickcall) [inline toolkits/springer/packages/springer-webfonts/HISTORY.md]: 2020 ??

Comment (davidpauljunior) [reply toolkits/springer/packages/springer-webfonts/HISTORY.md]: Oh. I wonder how many packages I've done that for 🤔

Comment (davidpauljunior) [reply toolkits/springer/packages/springer-webfonts/HISTORY.md]: At least the one before on this package it would seem

Comment (nickcall) [reply toolkits/springer/packages/springer-webfonts/HISTORY.md]: I probably signed it off

**Pull request merged**